Mathlib/Algebra/Group/WithOne/Basic.lean

```diff
@@ -6,6 +6,8 @@ Authors: Mario Carneiro, Johan Commelin
 import Mathlib.Algebra.Group.Equiv.Basic
 import Mathlib.Algebra.Group.WithOne.Defs
 import Mathlib.Data.Option.Basic
+import Mathlib.GroupTheory.GroupAction.Defs
+import Mathlib.Algebra.Group.Opposite
```

Comment:
Please try to keep imports sorted alphabetically.

Also note that `GroupTheory.GroupAction.Defs` was renamed to `Algebra.Group.Action.Defs` on master, so you should merge master and change that import.